src/databricks/labs/ucx/cli.py

```diff
@@ -34,10 +36,51 @@ def list_installations():
     print(json.dumps(all_users))
 
 
+def skip(**kwargs):
+    logger.info("Running skip command")
+    schema = kwargs["schema"]
+    table = kwargs["table"]
+    if not schema:
+        logger.error('"schema" is a required parameter.')
+        return None
+    ws = WorkspaceClient()
+    installation_manager = WorkspaceInstaller(ws)
+    logger.info("Fetching installation config.")
+    try:
+        sql_backend = StatementExecutionBackend(ws, installation_manager._current_config.warehouse_id)
+    except NotFound:
+        logger.error(
+            "Couldn't find UCX configuration in the user's home folder. "
+            "Make sure the current user has configured and installed UCX."
+        )
+        return None
+
+    if table:
+        try:
+            table_properties = sql_backend.fetch(f"SHOW TBLPROPERTIES hive_metastore.`{schema}`.`{table}`")
+        except BadRequest:
+            logger.error(f"{schema}.{table} wasn't found in this workspace")
```

Comment:
Log error details incase some other error generates `BadRequest`

Reply:
Also, is it expected to not find tables in a workspace? If so then it's not really an error.
If a table reaches this point and is not found then these failures should be counted.